channels.nix

```
@@ -39,6 +39,22 @@ rec {
       status = "rolling";
     };
 
+    "nixos-25.05" = {
+      job = "nixos/release-25.05/tested";
+      variant = "primary";
+      status = "beta";
+    };
+    "nixos-25.05-small" = {
+      job = "nixos/release-25.05-small/tested";
+      variant = "small";
+      status = "beta";
+    };
+    "nixpkgs-25.05-darwin" = {
+      job = "nixpkgs/nixpkgs-25.05-darwin/darwin-tested";
+      variant = "darwin";
+      status = "beta";
+    };
+    
```

I tried running `nix --extra-experimental-features 'nix-command flakes' flake check --all-systems` in order to check this PR. It gave me an error about trailing whitespace. I was able to fix the error by making this change:

```suggestion
    };

```

---

Run `nix fmt`.